includes/User.php

```diff
@@ -4194,7 +4105,7 @@ public static function crypt( $password, $salt = false ) {
 		// Wikia change - begin
 		// @see PLATFORM-2502 comparing new passwords in PHP code.
 		// @todo mech remove after the new password hashing is implemented (PLATFORM-2530).
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)